### PR TITLE
WIP:Add top level fields

### PR DESCRIPTION
**Pull request by kgudel**
This implementation currently suffers from an issue noted by @schbetsy in #724 . The file that is being used to get field data currently only has one field listed for each edit, even those that use multiple fields. This will require an update of that file to show all of the used fields, once that happens this code will need to be updated to separate the included fields into a list.

Addresses #717

### CODE REVIEW

File: Documents/API.md [hunk 1]
```diff
@@ -275,7 +289,8 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
         {
             "edits": [
                 {
-                   "edit": "q007",
+                   "edit": "Q007",
+                   "description: "Description of Q007",
```
Comment (awolfe76) [inline Documents/API.md]: You have the colon and quotes a bit mixed up.

`"description: "Description of Q007",`

should be 

`"description": "Description of Q007",`

File: Documents/API.md [hunk 2]
```diff
@@ -314,7 +336,14 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
     },
     {
       "edit": "V550",
+      "description: "Description of V550",
```
Comment (awolfe76) [inline Documents/API.md]: Same as above.

Comment (kgudel) [inline Documents/API.md]: Thanks, I'll review this and clean it up.

**Comment by schbetsy** [comment]
The front-end has updated what it wants to see for the format of the edits endpoint. The current version is in [this comment](https://github.com/cfpb/hmda-platform/issues/717#issuecomment-267459162) of #717.

**Comment by kgudel** [comment]
This is closed with updates to the format that the front-end wants.

**Pull request closed without merging**